### DISCUSSION TITLE
Nicknames and usernames on Harked

### DISCUSSION THREAD

**Discussion by MelonSlay** [Xvois/Harked#31] [Ideas]
Hey MelonSlay here, I was wondering if you could change the profile name feature. The username is the profile name instead of nickname, can you change this please? Lots of love from Pyongyang.

**Comment by Xvois** [comment]
I do not believe this is possilble with the data provided by the Spotify API. 

The username on Harked is set as the display name of the Spotify account, see [user profile information](https://developer.spotify.com/documentation/web-api/reference/get-current-users-profile).
